radicle-httpd/src/api/v1/projects.rs

```diff
@@ -663,7 +741,7 @@ mod routes {
                 "name": "hello-world",
                 "description": "Rad repository for tests",
                 "defaultBranch": "master",
-                "delegates": ["did:key:z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi"],
+                "delegates": [NID],
```

Comment:
```suggestion
                "delegates": [DID],
```
NIDs don't have the did:key: prefix :)